cassiopeia/data/CassiopeiaTree.py

```diff
@@ -2018,3 +2018,38 @@ def get_unmutated_characters_along_edge(
     def copy(self) -> "CassiopeiaTree":
         """Full copy of CassiopeiaTree"""
         return copy.deepcopy(self)
+
+    def impute_unambiguous_missing_states(self):
```

Comment:
Ah, good point.

I think `impute_missing_states` is a misleading name though, because not all missing states are being imputed: only those that are `unambiguously' imputable. E.g. in this tree with just one root and one leaf:

1 --> -1

the state -1 can be imputed unambiguously as 1. However, in the tree:

0 --> -1

the missing state cannot be imputed with 100% confidence.

How about we call this `impute_deducible_missing_states`?

Reply:
Renamed this to `impute_deducible_missing_states`